esdm/esdm_drng_mgr.c

```diff
@@ -645,13 +646,33 @@ void esdm_drng_force_reseed(void)
 static bool esdm_drng_must_reseed(struct esdm_drng *drng)
 {
 	struct timespec check_time = drng->last_seeded;
-	bool internal_bits_reached = atomic_read(&drng->internal_bits) >= (ESDM_DRNG_MAX_RESEED_BITS);
+	bool internal_bits_reached = (ESDM_DRNG_RESEED_THRESH_BITS > 0) && (atomic_read(&drng->internal_bits) >= ESDM_DRNG_RESEED_THRESH_BITS);
```

Comment:
I am slightly allergic to negative integers. 😀

Can I ask that we use uint32_t here? Thus when setting 0xffffffff, we effectively disable the check, too.
That said, please remove the check for <0 and change all instances to uint32_t.
Also, may I ask to set the meson_options.txt default value to0xffffffff?